src/Redux/api.tsx

```diff
@@ -171,10 +177,14 @@ const routes = {
 
   getPermittedFacility: {
     path: "/api/v1/facility/{id}/",
+    method: "GET",
+    TRes: Res<FacilityModel>(),
```

Comment:
Rebase with latest develop branch, the `Res` has been renamed to `Type` to make it generic for use with `TBody` too...